.github/workflows/ci-ros-lint.yml

```
@@ -33,7 +33,7 @@ jobs:
 
   ament_lint_100:
     name: ament_${{ matrix.linter }}
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Humble won't move with the new Ubuntu but this will break with the release of 24.04.
```suggestion
    runs-on: ubuntu-22.04
```
How about this?

---

sure, forgot to override this like I did in #392